ams/ams-server/src/main/java/com/netease/arctic/ams/server/optimize/BaseOptimizeCommit.java

```diff
@@ -204,11 +207,11 @@ private void minorCommit(ArcticTable arcticTable,
       overwriteBaseFiles.set(SnapshotSummary.SNAPSHOT_PRODUCER, CommitMetaProducer.OPTIMIZE.name());
       overwriteBaseFiles.validateNoConflictingAppends(Expressions.alwaysFalse());
       AtomicInteger addedPosDeleteFile = new AtomicInteger(0);
-      StructLikeMap<Long> oldPartitionMaxIds =
-          TablePropertyUtil.getPartitionMaxTransactionId(arcticTable.asKeyedTable());
       minorAddFiles.forEach(contentFile -> {
         // if partition min transactionId isn't bigger than max transactionId in partitionProperty,
         // the partition files is expired
+        StructLikeMap<Long> oldPartitionMaxIds =
```

Comment:
Why should we read the properties every time for a new file?

Reply:
It's no need. I moved it by mistake.